tests/discovery/test_hosts.py

```diff
@@ -1,10 +1,14 @@
 import requests_mock
 
+import pytest
+from netaddr import IPNetwork, IPAddress
+
```

Comment:
Please try to keep the following imports order:
1. built-in imports: `import os`
2. 3rd-party imports: `import netaddr`
3. this project imports: `import kube_hunter.conf`
4. empty line
5. built-in `from` imports: `from os import system`
6. 3rd-party `from` imports: `from netaddr import IPAddress`
7. this project imports: `from kube_hunter.conf import config`
8. relative imports (avoid those): `from . import some_function`

Remember to group `import X` statements and `from X import Y` separately.
TODO: document this in a developer guide

Reply:
Thanks for the quick-guide, didn't know that
according to that shouldn't it be like 
import requests_mock (3rd party)
import pytest (3rd party)
# empty line
from netaddr import IPNetwork, IPAddress (from 3rd party)

Reply:
Yes, followed by `from kube_hunter...` imports